### PR TITLE
Questionable messages 22

### PR DESCRIPTION
**Pull request by halla-ims**
Completed coding. Not as many unit tests as I would like. I wanted to add one for each case the error message is displayed, but that would take several days.

### CODE REVIEW

File: src/main/java/com/imsweb/mph/mpgroups/Mp2010HematopoieticGroup.java [hunk 1]
```diff
@@ -428,17 +429,21 @@ public TempRuleResult apply(MphInput i1, MphInput i2, MphComputeOptions options)
...
+        return MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(leftCode, rightCode, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(rightCode,
+                leftCode, year) ||
+                MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(leftCode, rightCode, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(
+                rightCode, leftCode, year);
     }
 
     private static boolean isAcuteToChronicTransformation(String earlierMorph, String latestMorph, int year) {
-        return MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(earlierMorph, latestMorph, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(latestMorph, earlierMorph,
+        return MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(earlierMorph, latestMorph, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(
+                latestMorph, earlierMorph,
```
Comment (bekeles) [inline src/main/java/com/imsweb/mph/mpgroups/Mp2010HematopoieticGroup.java]: The formatting doesn't look right here.

File: src/main/java/com/imsweb/mph/mpgroups/Mp2010HematopoieticGroup.java [hunk 2]
```diff
@@ -428,17 +429,21 @@ public TempRuleResult apply(MphInput i1, MphInput i2, MphComputeOptions options)
...
                 year);
     }
 
     private static boolean isChronicToAcuteTransformation(String earlierMorph, String latestMorph, int year) {
-        return MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(earlierMorph, latestMorph, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(latestMorph, earlierMorph,
+        return MphUtils.getInstance().getHematoDbUtilsProvider().isAcuteTransformation(earlierMorph, latestMorph, year) || MphUtils.getInstance().getHematoDbUtilsProvider().isChronicTransformation(
+                latestMorph, earlierMorph,
```
Comment (bekeles) [inline src/main/java/com/imsweb/mph/mpgroups/Mp2010HematopoieticGroup.java]: same here.

**Comment by halla-ims** [comment]
Formatting fixed and ready for review.

**Comment by bekeles** [comment]
closes #32

**Comment by bekeles** [comment]
@depryf Can you merge this?
Closes #22

**Pull request merged**